frontend/src/pages/dashboard/[id].tsx

```diff
@@ -188,6 +188,9 @@ export default function Dashboard() {
 
   useEffect(() => {
     (async () => {
+      if(router.isReady && workspaceId === "undefined"){
```

Comment:
This is a great start. Can you run prettier on this?

Reply:
@mv-turtle thoughts on showing a loader until router.isReady is true. I think without that we will see some UI flashing.

Reply:
Ah sure, my bad

Reply:
There's no difference even after running prettier on it. Do you want me to try something that may give the desired result?

Reply:
Hmm, that is strange. How are you running prettier:

For context, the above line should be formatted as:

```
if (router.isReady && workspaceId === "undefined") {
```

Reply:
Ah my bad, I forgot to add the node_modules path to prettier(since I switched to a new device). Pushing it with change